conversation_end.go

```diff
@@ -4,11 +4,18 @@ import (
 	"context"
 	"fmt"
 	"net/http"
+	"time"
 )
 
+type EndParams struct {
+	// Finished optionally defines the time when the conversation ended.
+	// If not given, this will default to the current time.
+	Finished *time.Time `json:"finished,omitempty"`
+}
+
 // EndConversation ends a conversation.
-func (c *Client) EndConversation(ctx context.Context, conversationID string) error {
-	rsp, err := c.makeRequest(ctx, http.MethodPost, fmt.Sprintf("/conversations/%s/end", conversationID), nil)
+func (c *Client) EndConversation(ctx context.Context, conversationID string, p EndParams) error {
+	rsp, err := c.makeRequest(ctx, http.MethodPost, fmt.Sprintf("/conversations/%s/end", conversationID), p)
```

Comment:
See my comments in the backend PR 😉 

I think this should be a `PUT` request instead.